engine/lib/phases/phase_reject.ml

```diff
@@ -41,6 +41,45 @@ end
 
 module _ (FA : Features.T) : Phase_utils.PHASE = Continue (FA)
 
+module Break (FA : Features.T) = struct
+  module FB = struct
+    include FA
+    include Features.Off.Break
+  end
+
+  include
+    Feature_gate.Make (FA) (FB)
+      (struct
+        module A = FA
+        module B = FB
+        include Feature_gate.DefaultSubtype
+
+        let break = reject
+        let metadata = make_metadata Break
+      end)
+end
+
+module QuestionMark (FA : Features.T) = struct
+  module FB = struct
+    include FA
+    include Features.Off.Question_mark
+  end
+
+  include
+    Feature_gate.Make (FA) (FB)
+      (struct
+        module A = FA
+        module B = FB
+        include Feature_gate.DefaultSubtype
+
+        let question_mark = reject
+        let metadata = make_metadata QuestionMark
+      end)
+end
+
+module _ (FA : Features.T) : Phase_utils.PHASE = Continue (FA)
+module _ (FA : Features.T) : Phase_utils.PHASE = Break (FA)
```

Comment:
Ideally those should be ensured statically by the phase removing breaks and question marks